Makefile

```diff
@@ -18,7 +18,7 @@ setup: setup-git ## Setup Hugo locally
 	npm install postcss-cli autoprefixer postcss
 
 gen-components: ## Generate components markdown from baker@latest
```

Comment:
What about the comment?

Reply:
yeah. good point. @lucarin91 once you fix the comment we can merge this